iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -754,5 +754,25 @@ - (void)resetElevationMyPositionChanged {
   self.bm.SetElevationMyPositionChangedCallback(nullptr);
 }
 
+- (dp::Color)getColorFromUIColor:(UIColor *)color {
+    CGFloat fRed = 0;
+    CGFloat fGreen = 0;
+    CGFloat fBlue = 0;
+    CGFloat fAlpha = 0;
+
+    [color getRed:&fRed green:&fGreen blue:&fBlue alpha:&fAlpha];
+
+    const uint8_t red = [self convertColorComponentToHex:fRed];
+    const uint8_t green = [self convertColorComponentToHex:fGreen];
+    const uint8_t blue = [self convertColorComponentToHex:fBlue];
+    const uint8_t alpha = [self convertColorComponentToHex:fAlpha];
+
+    return dp::Color(red, green, blue, alpha);
+}
+
+- (uint8_t)convertColorComponentToHex:(CGFloat)color {
+    static constexpr uint8_t kMaxChannelValue = 255;
+    return (uint8_t)(color * kMaxChannelValue);
```

Comment:
1. No need to cast to (uint8_t).
2. If there is a need (compiler warning), then use `static_cast<type>(xxx)` instead.